### DISCUSSION TITLE
TinyEngine的浏览器兼容性

### DISCUSSION THREAD

**Discussion by BentleyGao** [opentiny/tiny-engine#450] [General]
请问下TinyEngine的支持谷歌浏览器多少版本的，能否具体列下版本号，感谢
另外是否兼容火狐浏览器

**Comment by chilingling** [comment]
整体设计器核心：
- chrome 93+
- firefox 108+
- 不支持 IE浏览器

部分插件在 firefox 无法支持，比如下载代码的插件使用到了 `window.showDirectoryPicker` api，但是到目前为止 firefox 还未支持，如果需要在 firefox 使用该插件，建议使用下载文件等 api 进行替换。

**Reply by BentleyGao** [reply]
感谢！
